test/sequential/test-inspector-port-cluster.js

```diff
@@ -287,9 +288,9 @@ function masterProcessMain() {
 
         cluster.setupMaster(clusterSettings);
 
-        assert.throws(() => {
+        common.expectsError(() => {
```

Comment:
As a recommendation for the future: `assert.throws` is actually becoming more and more powerful and using it is probably better than using `common.expectsError` (even though that is still useful as a callback function).

Reply:
What should I do to make it work with `assert.throws`?

This is what I get:
```
AssertionError [ERR_ASSERTION]: type: expected [Function: RangeError], not undefined
    at masterProcessMain (/home/mzasso/git/nodejs/node/test/sequential/test-inspector-port-cluster.js:238:16)
    at Object.<anonymous> (/home/mzasso/git/nodejs/node/test/sequential/test-inspector-port-cluster.js:347:3)
```

Reply:
Right now there are some special handled properties in `common.expectsError` that do not exist in `assert.throws`. One is the `message` property: it is either a strict equal comparison or you can use a regular expression. Using regular expressions is not (yet) possible (I am still considering adding options to accept those as well) with `assert.throws`.
The other difference is the `type` property: it will actually check for the specific error type. That is not possible at all with `assert.throws` and instead you can check for the `name` property.